src/strings.cpp

```diff
@@ -986,26 +994,14 @@ static void FormatString(StringBuilder &builder, const char *str_arg, StringPara
 						sub_args.SetParam(i++, param);
 					} else {
 						s++; // skip the leading \"
-						char *g = stredup(s);
-						g[p - s - 1] = '\0'; // skip the trailing \"
-
-						sub_args_need_free[i] = true;
-						sub_args.SetParam(i++, (uint64)(size_t)g);
-						_game_script_raw_strings.push_back(g);
+						sub_args.SetParam(i++, std::string(s, p - s - 1)); // also skip the trailing \".
```

Comment:
```suggestion
						sub_args.SetParam(i++, std::string(s, p - s)); // also skip the trailing \".
```
Last char vs size I think.